en/frames_multicopter/qav_r_5_kiss_esc_racer.md

```diff
@@ -0,0 +1,218 @@
+# QAV-R 5" KISS ESC Racer (Pixracer)
```

Comment:
@MaEtUgR I'm migrating this doc from the DevGuide into PX4 user guide (I'll merge shortly). Are there any things that you see that need to be updated?

For example, you talk about using master build for various reasons, but I suspect that is no longer needed, and it might be worth making this "current".

If this is not "your doc" are you able to suggest who contributed it?

Reply:
@hamishwillee Thanks for cleaning up and finally moving things to the user guide 👍 
I'm the right one to blame, I did the build and I documented it. Sorry, I was on holiday...
- My biggest question is: Why was the video moved to the bottom?
- It's good that you mention the included branch descripion. The very important feature ist still not available on the stable branch or any release yet. But fell free to switch the wording to what you feel is an appropriate way to say "use the master branch until version 1.7 will be released".

Reply:
Hi @MaEtUgR, you're welcome! I added note that the relevant code will appear in PX4v1.7 - we can try to remember to update when that comes out.  

> My biggest question is: Why was the video moved to the bottom?

Consistency. I usually try to have one image up the top of the page showing off the vehicle. A video would be much better but across all of our docs, the videos are often awful -  so I tend to use still up top and have a "Video" section. 

This one is a pretty good video. I could be inconsistent, use the video, and have the image at the bottom if you really prefer under "Gallery" heading.

Reply:
@hamishwillee I would really appreciate if you could make an exception and place the video on top.
I think it shows off really well what's currently possible with PX4 despite some people thinking of this flight stack to be designed for large multicopters with rather slow, automatic navigation.
Also it gives me and hopefully also others the motivation to record and cut further useful videos for such build logs.

Reply:
Sounds reasonable! Done.